dspace-server-webapp/src/test/java/org/dspace/app/rest/authorization/GenericAuthorizationFeatureIT.java

```diff
@@ -209,214 +209,214 @@ private void testAdminsHavePermissionsAllDso(String feature) throws Exception {
         String siteId = ContentServiceFactory.getInstance().getSiteService().findSite(context).getID().toString();
 
         // Verify the general admin has this feature on the site
-        getClient(adminToken).perform(get("/api/authz/authorizations/search/object?embed=feature&uri="
+        getClient(adminToken).perform(get("/api/authz/authorizations/search/object?size=1000&embed=feature&uri="
```

Comment:
Why is it necessary to add `size=1000` to every single `/api/authz/authorizations/search/object` request?  Is this now a required parameter?  If so, this could impact the User Interface & we would need to document this in our REST Contract.